apollo-runtime/api/apollo-runtime.api

```diff
@@ -377,12 +377,6 @@ public final class com/apollographql/apollo3/network/ws/GraphQLWsProtocol$Factor
 	public fun getName ()Ljava/lang/String;
 }
 
-public final class com/apollographql/apollo3/network/ws/OkHttpWebSocketEngine : com/apollographql/apollo3/network/ws/WebSocketEngine {
-	public fun <init> ()V
-	public fun <init> (Lokhttp3/WebSocket$Factory;)V
-	public fun open (Ljava/lang/String;Ljava/util/Map;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
-}
```

Comment:
Can we keep this public in case someone wants to customize their `WebSocket.Factory`. Also for symmetry with `OkHttpEngine`. Then users can use them from `ApolloClient.Builder.webSocketEngine()`

Reply:
Not sure why I thought I saw the apple one was internal, but no. Reverting 👍